File: src/gestorAplicacion/logistica/CargaExtra.java
package gestorAplicacion.logistica;
import java.io.*;

import java.util.ArrayList;

public abstract class CargaExtra implements Serializable{
    
    private static final long serialVersionUID = 1L;
    private int precio;
    private String tamaño;
    private  static ArrayList<CargaExtra> cargas = new ArrayList<CargaExtra>();

    //constructores
    public CargaExtra(int precio, String tamaño) {
        this.precio = precio;
        this.tamaño = tamaño;
        cargas.add(this);
    }


     //getter and setter
    public int getPrecio() {
        return precio;
    }
    public void setPrecio(int precio) {
        this.precio = precio;
    }
    public String getTamaño() {
        return tamaño;
    }
    public void setTamaño(String tamaño) {
        this.tamaño = tamaño;
    }
    public static ArrayList<CargaExtra> getCargas() {
        return cargas;
    }
    public static void setCargas(ArrayList<CargaExtra> cargas) {
        CargaExtra.cargas = cargas;
    }
    public static long getSerialversionuid() {
        return serialVersionUID;
    }
}

File: Python/src/gestorAplicacion/logistica/CargaExtra.py
class CargaExtra:
    cargas = []
    def __init__(self,precio,tamaño):
        self.precio = precio
        self.tamaño = tamaño
        CargaExtra.cargas.append(self)
    def agregar(self):pass
    
    def getPrecio(self):
        return self.precio
    def setPrecio(self,precio):
        self.precio = precio
    def getTamaño(self):
        return self.tamaño
    def setTamaño(self,tamaño):
        self.tamaño = tamaño
    @classmethod
    def getCargas(cls):
        return cls.cargas
    @classmethod
    def setCargas(cls,cargas):
        cls.cargas = cargas